samples/net/sockets/big_http_download/prj.conf

```diff
@@ -2,7 +2,8 @@
 CONFIG_NEWLIB_LIBC=y
 CONFIG_MBEDTLS=y
 CONFIG_MBEDTLS_ENABLE_HEAP=y
-CONFIG_MAIN_STACK_SIZE=2176
+CONFIG_MAIN_STACK_SIZE=2180
```

Comment:
We are only increasing the stack size for 4 bytes. Is this really enough here, I mean that if something changes in the system (for example not related to this sample), then we might run out of stack again. I suggest that the value is increased more, like to 2256 for example (I just picked some higher random value that is dividable by 8)

Reply:
I've done a mistake in testing previously. Tested it now with `CONFIG_MAIN_STACK_SIZE=2176` and it works fine on my nucleo-h745zi-q board. Thus this change was reverted.